style.css

```diff
@@ -0,0 +1,336 @@
+}
+
+.header__menu-link:hover {
+  color: #e51937;
+}
+
+.header__menu-link:hover::after {
+  opacity: 1;
+}
+
+#light {
+  color: #e51937;
+  margin: 30px 50px 0 0;
+}
+
+.header__line {
+  position: absolute;
+  left: 458px;
+  top: 50px;
+}
+
+.menu__icon {
+  width: 24px;
+  height: 15px;
+  margin: 0 5px;
+}
+
+.main__title {
+  font-size: 90px;
+  font-family: 'montserratsemibold', sans-serif;
+  margin: 40px 0 0 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__subtitle {
+  font-size: 15px;
+  font-family: 'montserratmedium', sans-serif;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__pic {
+  width: 100%;
+  margin: -110px 0 0 0;
+}
+
+.main__info {
+  display: flex;
+  justify-content: center;
+}
+
+.info__content {
+  display: flex;
+}
+
+.main__content {
+  margin: 30px 0;
+}
+
+.main__content-item {
+  margin: 0 20px;
+  display: flex;
+}
+
+.content {
+  display: flex;
+  align-items: center;
+  margin: 0 20px;
+}
+
+.content__item {
+  margin-right: 5px;
+  display: flex;
+  flex-direction: column;
+}
+
+.content__pic {
+  width: 46px;
+  height: 35px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.content__item-one {
+  font-size: 35px;
+  font-family: 'montserratsemibold', sans-serif;
+  margin: 0 5px 5px 0;
+  display: flex;
+  align-items: baseline;
+  text-align: left;
+}
+
+.content__one-big {
+  font-size: 35px;
+  font-family: 'montserratsemibold', sans-serif;
+  margin: 0 5px 0 0;
+}
+
+.content__one-small {
+  font-size: 20px;
+  font-family: 'montserratsemibold', sans-serif;
+}
+
+.content__item-two {
+  font-size: 12px;
+  font-family: 'montserratregular', sans-serif;
+}
+
+.button__out {
+  border: none;
+  display: flex;
+  align-items: center;
+}
+
+button:hover {
+  opacity: 0.5;
+}
+
+.button__pic {
+  margin-right: 5px;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.button__redwheel {
+  display: none;
+}
+
+.button__border {
+  display: flex;
+  align-items: center;
+  border: 2px solid #e51937;
+  color: #e51937;
+}
+
+.button__text {
+  text-transform: uppercase;
+}
+
+.main__pointer {
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+section {
+	
+}
+
+.section__logo {
+	position: absolute;
+}
+
+.section__title {
+  color: #e51937;
+  font-size: 70px;
+  font-family: 'montserratsemibold', sans-serif;
+  position: relative;
+  
+}
+
+.article {
+ position: relative;
+  height: 100%;
+ left: 69%;
+ width: 31%;
+  color: white;
+  background-color: #e51937;
+
+}
+
+.article__info {
+  padding: 40px 30px;
+}
+
+.article__charset {
+	margin: 30px 0 100px 0;
+	display: flex;
+	flex-direction: column;
+	padding: 0 25px;
+}
+
+.article__top {
+   padding: 10px 35px;
+}
+
+.article__title {
+	font-size: 30px;
+}
+
+
+.charset {
+	display: flex;
+	padding: 30px 0;
+	
+}
+
+.charset__item {
+	margin: 0 10px;	
+}
+
+.charset__item-one {
+	display: flex;
+	align-items: baseline;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)